### ISSUE TITLE
Translations from English to Swedish

### ISSUE THREAD

**Issue by Annfo637** [infor-design/enterprise#5987]
**Describe the bug**
One translation should be updated and one translation doesn't seem to work at all.

**To Reproduce**

1. In Portal V2/Homepages, change language to Swedish and add the Notes (Anteckningar) widget.
2. Click on the ellipse (...) and choose "Hyperlänk".
3. The heading in the dialog should be "länk" instead of "fästpunktslänk". [New window] doesn't seem to be translated at all.

**Expected behavior**
As above, "fästpunktslänk" should be "länk", also "[New Window]" should be translated to "Nytt fönster".

**Version**

- ids-enterprise: 4.56.0

**Screenshots**
![image](https://user-images.githubusercontent.com/54262041/148034126-16fb3833-4b81-47a3-9039-7f7a11f2b192.png)

**Platform**
 - Infor Application/Team Name: Portal V2
 - OS Version: Windows 10
 - Browser Name: Chrome
 - Browser Version: 95.0.4638.69 (Official Build) (64-bit)

**Comment by tmcconechy** [comment]
cc @hwondergem can you check this one as well? Thanks!

**Comment by tmcconechy** [comment]
Also @hwondergem i sent an invite https://github.com/orgs/infor-design/people/pending_invitations just so i can assign issues to you. Let me know if you can see it or check your email attached?

**Comment by hwondergem** [comment]
@tmcconechy, @Annfo637 Feedback from the linguist: 
The translation of “anchor” comes from MS Language Portal, our primary source for IT terminology. Changing the translation to “länk” creates a conflict with the translation of “Insert Link”. According to the comments, “Insert Anchor” means “Insert a hyperlink in an editor, so maybe “anchor” should be translated as “hyperlänk” instead? (In that case, why use “anchor” in the source?)
 
![image](https://user-images.githubusercontent.com/67783011/148210805-12bdd376-f544-4df2-a0dd-a098b1f304d5.png)


The string [New window] does not exist in IDS strings file, only New Window without the brackets and with an uppercase W.

**Comment by tmcconechy** [comment]
Anchor (`a`) tag is synonymous with hyperlink in english. But we could easily change it to InsertHyperlink if that would help? I think for translation i guess you would call it whatever of the two makes the most sense? In case you want us to change it i added an additional string to nlbavwmemoq2/cc/FromDev/SohoUX/

The string is NewWindow the brackets indicate something wasnt found in the translations but i cannot see that here. So i assume something wrong on the Dev teams end @Annfo637 

https://main-enterprise.demo.design.infor.com/components/editor/example-index.html?locale=sv-SE
![Screen Shot 2022-01-05 at 9 21 57 AM](https://user-images.githubusercontent.com/814283/148233207-595e3c73-1e56-47aa-9612-a537ba699305.png)

**Comment by hwondergem** [comment]
@tmcconechy, if the concepts of anchor and hyperlink are synonymous, maybe using one is less confusing. 
However, according to [Techopedia](https://www.techopedia.com/definition/1947/anchor): 

> An anchor is an HTML tag that identifies a link to text or an image on the same page or to another specific location. An anchor can be used in two ways:
> 
> By using the “href” attribute to create a link to another page
> By using the "name" attribute to create a bookmark within a page
> The hyperlink text on which the user clicks is known as the anchor text. Anchor text is also known as link text or a link label.

**Comment by tmcconechy** [comment]
@hwondergem yep thats also my understanding. So technically this string is used to insert an anchor tag in the editor https://main-enterprise.demo.design.infor.com/components/editor/example-index.html but refering to it as a link/hyperlink might be more "user friendly". So i guess whichever is the best for translators?

**Comment by hwondergem** [comment]
OK @tmcconechy, in case the following are synonymous, update to comment: 'Insert a hyperlink in an editor, synonymous to hyperlink' for the first and third so translators can decide how to translate?
  InsertAnchor: { id: 'InsertAnchor', value: 'Insert Anchor', comment: 'Insert a hyperlink in an editor' },
  InsertHyperlink: { id: 'InsertHyperlink', value: 'Insert Hyperlink', comment: 'Insert a hyperlink in an editor' },
  (...)
  InsertLink: { id: 'InsertLink', value: 'Insert Link', comment: 'Insert Link in an editor' },

**Comment by tmcconechy** [comment]
The only thing is i have to pick one of those three to actually use. I cant use them all. So maybe it can be..

```
 InsertHyperlink: { id: 'InsertHyperlink', value: 'Insert Hyperlink', comment: 'Insert a hyperlink (synonymous with link or anchor tag) into an editor ' },
```

**Comment by hwondergem** [comment]
Yes, that's the best solution! Thanks.

**Comment by SofiK** [comment]
Hi, "New window" should be "Nytt fönster" in Swedish. "Nästa" means "Next".

**Comment by tmcconechy** [comment]
Seems to be incorrect here https://github.com/infor-design/enterprise/blob/main/src/components/locale/cultures/sv-SE.js#L346 - but if you fixed it in the next version i dont have yet then feel free to ignore.

**Comment by asandberg2** [comment]
Yes, it is fixed in the next version:

<img width="626" alt="image" src="https://user-images.githubusercontent.com/97168564/148784092-fabef98f-8d4c-4eb6-ade5-246d6dcc6816.png">

**Comment by jbrcna** [comment]
working as expected. moving this ticket to Done. thanks
v4.60.0-dev
https://main-enterprise.demo.design.infor.com/components/editor/example-index.html?locale=sv-SE
<img width="440" alt="image" src="https://user-images.githubusercontent.com/56722184/152470780-1ccb5a92-1f7b-455e-92e1-8dfc187f35d7.png">

**Comment by asandberg2** [comment]
Although this issue is closed, I would like to make sure that everything is clear regarding the translation of "Insert Anchor". In the latest translation update, a new string for "Insert Hyperlink" has been included, in addition to "Insert Anchor". Both have the same comment "Insert a hyperlink in an editor". Should both strings really be translated as "Insert Hyperlink"? Why have two different strings in English if the word for "anchor" should not be used in the localized version? Sorry, but this is confusing. I have currently translated "anchor" to "hyperlänk", but I don't think that it is a good solution.

**Comment by tmcconechy** [comment]
I talked to the translators and they wanted to call it "Link" in English and that would be easiest to translate apparently.
So thats what it is now.

It may take some time for the translators to translate the new string (month usually) but should be solved when that is done. Also have to assume providing that they translate "link" to something you would agree with but thats their decision as i've seen "hyperlänk" and "länk". 

For backwards compatibility i do still have the old strings for hyperlink and anchor but they aren't used after this anymore.

**Comment by asandberg2** [comment]
OK, if the old string "Insert Anchor" will not be used anymore, I will translate "anchor" as anchor (fästpunkt) in the Swedish translation and not as hyperlink, to avoid any confusion if "anchor" reappears somewhere else in future updates. Thank you!
